pkgs/development/compilers/flasm/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, unzip, bison, flex, gperf, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "flasm";
+  version = "1.64";
+
+  src = fetchurl {
+    url = "https://www.nowrap.de/download/flasm16src.zip";
+    sha256 = "0n7sqv0b4f3wjfrhlv9d1ynza1chcmnrqxvpp0b3dh68ridnjdxv";
+  };
+
+  unpackPhase = ''
+    unzip $src
```

Comment:
Can we use fetchzip to do this automatically?

Reply:
Done.